src/main/kotlin/no/item/xp/plugin/renderers/ts/TSRenderUtils.kt

```diff
@@ -36,6 +36,7 @@ fun renderComment(comment: String?, indentLevel: Int): String {
 
 fun getInterfaceName(nameWithoutExtension: String): String {
   return nameWithoutExtension
+    .split(".")[0]
```

Comment:
Since we in 2021 still are dependent on FilenameUtils to get `baseName` and without this interface names will end with `.d`